README.md

```diff
@@ -4,6 +4,41 @@ Oaken is fixtures + factories + seeds for your Rails development & test environm
 
 You may want to head straight to the [examples](examples) directory.
 
+## Benefits
+
+- In development:
+  - Add conventions for `db/seeds.rb` that feel like Rails.
+  - Grouping by scenario: choose how to best group your data.
+  - Reveal your Domain Model: describe your object graph sequentially, helping developers see how your app works.
+  - Ruby-based recipe-like data scripts: setup accounts first, then attach users, then hang other models off of those.
+  - Raise modelling problems: Oaken can help expose when something feels awkward or off to give you design clues.
+
+- In tests, reuse development seed data:
+  - Tests mirror what developers have already seen in their dev browser.
+  - Easier developer onboarding by building a proper mental model faster.
+  - Less code that's easier to maintain with more visibility into your object graph.
+
+### Benefits over factories
+
+- Fast tests! Seed shared records once and reuse them across tests, transactions rollback changes.
+- Sets you up for the long term: factories drain your mental resources quicker 
+- Visibility into your system: factories optimize for isolation, which make it easy to get started, but becomes more complex over time.
```

Comment:
Also worth noting that you DRY up your tests _more_ for non-trivial systems because you write the Oaken seed for common cases _once_, rather than per-test/file